src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -256,7 +256,7 @@ float FixedwingAttitudeControl::get_airspeed_and_update_scaling()
 	 *
```

Comment:
The comment above should be updated to reflect the new logic based on "stall" speed

Reply:
true, thanks for pointing it out.